File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_Three/Java/prog3.java
class Demo {
	public static void main(String[] args) {
		int num1 = 1;
		for(int i = 0; i < 4; i++) { 
			for(int j = 0; j <= i; j++) {
				System.out.print(num1 + " ");
				num1++;
			}
			System.out.println();
		}
	}
}

File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_Three/C++/prog3.cpp
#include<iostream>

int main() {
	int num1 = 1;
	for(int i = 0; i < 4; i++) {
		for(int j = 0; j <= i; j++) {
			std::cout << num1 << " ";
			num1++;
		}
		std::cout << std::endl;
	}
	return 0;
}
